kiwmi/luak/kiwmi_keyboard.c

```diff
@@ -16,6 +16,29 @@
 #include "input/keyboard.h"
 #include "luak/kiwmi_lua_callback.h"
 
+static int
+l_kiwmi_keyboard_configure(lua_State *L)
+{
+    struct kiwmi_keyboard *keyboard =
+        *(struct kiwmi_keyboard **)luaL_checkudata(L, 1, "kiwmi_keyboard");
+
+    struct xkb_rule_names rules = {
```

Comment:
I feel like this could do with default values (empty strings or NULL from the docs) so that the user cal leave out stuff, similar to setxkbmap, which is way easier to use (`keyboard:configure("", "", "de", "", "")` makes me remember the position of something and be aware of the rest for no reason).

Reply:
Also I feel like a table would be easier to use (remembering the position of 5 arguments, especially when to most people 3 or 4 are alien isn't optimal).

Reply:
Also I'd rather call this `names`--`rules` is only of the fields afterall.

Reply:
Yeah I think about a lua table to use is more convenient, so I give that a try.
I also change the variable name